sst/nights/facilities_temperatures_reports.ipynb

```diff
@@ -0,0 +1,442 @@
+{
```

Comment:
<p>Optional really: add more formatting to the output table, so that the time column is Time (UTC) and the temperatures are C, with two significant figures. Same for the other cells below.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/reports-performance-summary/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>